### PR TITLE
Feature/isdb 364

### PR DESCRIPTION
**Pull request by RonanCog**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/test/resources/request/bulk-scan-fl401-transform-input.json [hunk 1]
```diff
@@ -265,110 +263,6 @@
       "name":"stopRespondentFromDoing_1",
       "value":"Yes"
     },
-    {
```
Comment (parvezhmcts) [inline src/test/resources/request/bulk-scan-fl401-transform-input.json:268]: why did you remove the below below objects?

File: src/test/resources/response/bulk-scan-fl401-transform-output.json [hunk 2]
```diff
@@ -39,25 +83,13 @@
         }
       ],
       "fl401ApplicantTable": {
-        "firstName": "testfirstname",
```
Comment (parvezhmcts) [inline src/test/resources/response/bulk-scan-fl401-transform-output.json:42]: why did you remove all below objects?

File: src/test/resources/response/bulk-scan-fl401-transform-output.json [hunk 3]
```diff
@@ -39,25 +83,13 @@
...
-          "AddressLine1": "No",
-          "AddressLine2": "applicant second line",
-          "PostTown": "No",
-          "County": "applicant county",
-          "PostCode": "cv2 4nz"
-        },
-        "phoneNumber": "No",
-        "email": "No",
-        "preferToBeContacted": "No"
+        "firstName": null,
```
Comment (parvezhmcts) [inline src/test/resources/response/bulk-scan-fl401-transform-output.json:86]: why these objects replaced with null value?

File: src/test/resources/response/bulk-scan-fl401-transform-output.json [hunk 4]
```diff
@@ -200,21 +255,19 @@
       ],
       "fl401SolicitorDetailsTable": {
         "solicitorOrg": {
-          "OrganisationName": "solicitor firm name"
+          "OrganisationName": null
         },
         "address": {
```
Comment (parvezhmcts) [inline src/test/resources/response/bulk-scan-fl401-transform-output.json:260]: why did you replace objects value with null.

**Review by parvezhmcts — COMMENTED** [review]
Please check and incorporate review comments.

**Review by hmcts-kodatala — COMMENTED** [review]
Pls check review comments and correct all

File: src/main/java/uk/gov/hmcts/reform/bulkscan/constants/BulkScanConstants.java [hunk 5]
```diff
@@ -335,6 +335,54 @@ public final class BulkScanConstants {
     public static final String APPLICANT_MARITAL_STATUS = "applicantMaritalStatus";
     // End of A58
```
[the remark this answers is not in the document]
Comment (RonanCog) [reply src/main/java/uk/gov/hmcts/reform/bulkscan/constants/BulkScanConstants.java:337]: done

File: src/main/java/uk/gov/hmcts/reform/bulkscan/services/BulkScanFL401ConditionalTransformerService.java [hunk 6]
```diff
@@ -279,6 +321,120 @@ private List buildTransformChild(
...
+        childInput.add(row4);
+
+        for (String input : childInput) {
+            LinkedTreeMap<String, String> childDetails = new LinkedTreeMap<>();
+
+            final LinkedTreeMap<String, LinkedTreeMap<String, String>> childrenLinkedTreeMap =
+                    new LinkedTreeMap();
+
+            if (null != input) {
+                final String[] columnDetails = input.split(",");
```
[the remark this answers is not in the document]
Comment (RonanCog) [reply src/main/java/uk/gov/hmcts/reform/bulkscan/services/BulkScanFL401ConditionalTransformerService.java:363]: done

**Review by hmcts-kodatala — APPROVED** [review]
OK

**Review by Murshadhaque — APPROVED** [review]

**Review by Shilpa-Krishna — APPROVED** [review]
